codalib/bagatom.py

```diff
@@ -89,10 +89,7 @@ def getBagTags(bagInfoPath):
     get bag tags
     """
 
-    try:
-        bagInfoString = open(bagInfoPath, "r").read().decode('utf-8')
-    except UnicodeDecodeError:
-        bagInfoString = open(bagInfoPath, "r").read().decode('iso-8859-1')
+    bagInfoString = open(bagInfoPath, "r").read()
```

Comment:
I don't think we can wholly throw out considering encodings here. I believe if you try to read a file with a 'iso-8859-1' encoding here, it could still throw a UnicodeDecodeError. I am submitting a test to show the failure.